<launch>

 <!-- <node pkg="nodelet" type="nodelet" name="imu_manager" 
args="manager" output="screen" />-->

  <node pkg="ultron_kernel" type="mpu9250_serial_to_imu_node" name="mpu9250_serial_to_imu_node" required="true">
    <param name="port" value="/dev/ttyUSB0"/>
    <remap from="joy" to="joy_teleop/joy" /> 
  </node>

<!--
<node pkg="imu_complementary_filter" type="complementary_filter_node"
      name="complementary_filter_gain_node">
    <param name="do_bias_estimation" value="true"/>
    <param name="do_adaptive_gain" value="true"/>
    <param name="use_mag" value="false"/>
    <param name="gain_acc" value="0.01"/>
    <param name="gain_mag" value="0.01"/>
 
  
</node>
-->
<!--
<node pkg="imu_filter_madgwick" type="imu_filter_node"
      name="imu_filter_node" output="screen">
    <param name="do_bias_estimation" value="true"/>
    <param name="do_adaptive_gain" value="true"/>
    <param name="use_mag" value="false"/>
    <param name="gain_acc" value="0.01"/>
    <param name="gain_mag" value="0.01"/>
 
  
</node>
-->
</launch>
